data/top_model/abp_output_transform.txt

```diff
@@ -0,0 +1,108 @@
+
+00:00:49:000             20                 out            subnet1
+
+00:00:59:000              0                 out          receiver1
+
+00:01:02:000              0                 out            subnet2
+
+00:01:02:000              0      ackReceivedOut            sender1
+
+00:01:12:000              3       packetSentOut            sender1
+
+00:01:12:000             31             dataOut            sender1
+
+00:01:15:000             31                 out            subnet1
+
+00:01:25:000              1                 out          receiver1
+
+00:01:28:000              1                 out            subnet2
+
+00:01:28:000              1      ackReceivedOut            sender1
+
+00:01:30:000              4                 out      generator_con
+
+00:01:40:000              4       packetSentOut            sender1
+
+00:01:40:000             40             dataOut            sender1
+
+00:01:43:000             40                 out            subnet1
+
+00:01:53:000              0                 out          receiver1
+
+00:01:56:000              0                 out            subnet2
+
+00:01:56:000              0      ackReceivedOut            sender1
+
+00:02:06:000              5       packetSentOut            sender1
+
+00:02:06:000             51             dataOut            sender1
+
+00:02:09:000             51                 out            subnet1
+
+00:02:19:000              1                 out          receiver1
+
+00:02:22:000              1                 out            subnet2
+
+00:02:22:000              1      ackReceivedOut            sender1
+
+00:20:00:000              3                 out      generator_con
+
+00:20:10:000              1       packetSentOut            sender1
+
+00:20:10:000             11             dataOut            sender1
+
+00:20:13:000             11                 out            subnet1
+
+00:20:23:000              1                 out          receiver1
+
+00:20:26:000              1                 out            subnet2
+
+00:20:26:000              1      ackReceivedOut            sender1
+
+00:20:36:000              2       packetSentOut            sender1
+
+00:20:36:000             20             dataOut            sender1
+
+00:20:39:000             20                 out            subnet1
+
+00:20:49:000              0                 out          receiver1
+
+00:20:52:000              0                 out            subnet2
+
+00:20:52:000              0      ackReceivedOut            sender1
+
+00:21:02:000              3       packetSentOut            sender1
+
+00:21:02:000             31             dataOut            sender1
+
+00:21:32:000              3       packetSentOut            sender1
+
+00:21:32:000             31             dataOut            sender1
+
+00:21:35:000             31                 out            subnet1
+
+00:21:45:000              1                 out          receiver1
+
```

Comment:
There should not be an empty lines between two rows

Reply:
Thanks for your valuable comments. I fixed it. How do you want me to push these changes to GitHub? Shall I push it to the same branch?